pax-runtime/src/layout.rs

```diff
@@ -31,13 +32,26 @@ pub fn compute_tab(
         container_bounds.untyped(),
         cp_width.untyped(),
         cp_height.untyped(),
+        size_fallback.untyped(),
     ];
 
     let bounds = Property::computed_with_name(
         move || {
             let p_bounds = cp_container_bounds.get();
-            let width = cp_width.get().evaluate(p_bounds, Axis::X);
-            let height = cp_height.get().evaluate(p_bounds, Axis::Y);
+            let (fallback_width, fallback_height) = match size_fallback.get() {
+                Some((x, y)) => (Some(x), Some(y)),
```

Comment:
is there a reason we combine them into a tuple in the first place? Why don't we just have rendered_width and rendered_height as two separate values the whole time? They arrive that way from the chassis.

Reply:
They always come back from the chassi as a tuple, ie either both exist or none of them. I could split them appart earlier, do you mean as two separate fields on expanded_node?

Reply:
*chassis lol

Reply:
Actually this works for now; given that we'll always be sending both back regardless of what's being listened to.